### PR TITLE
Github Action to build docker image of knative mkdocs

### PR DESCRIPTION
**Pull request by kvmware**
Fixes https://github.com/knative/docs/issues/4449
## Proposed Changes 
- Github action to build and push latest commit to dockerhub

### CODE REVIEW

File: .github/workflows/knative-mkdocs-dockerimage.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          file: ./hack/docker/Dockerfile
+          push: true
+          tags: krsnam/mkdocs:latest
```
Comment (kvmware) [inline .github/workflows/knative-mkdocs-dockerimage.yaml:24]: tags should point to the community's dockerhub repo instead of mine.

Comment (csantanapr) [reply .github/workflows/knative-mkdocs-dockerimage.yaml:24]: knative community doesn't use dockerhub image registry

We use Prow, Prow has the credentials to push to the Knative google container registry where we have an account and store all the nightly builds.

cc @evankanderson

**Comment by kvmware** [comment]
Will add the images to the repo as shown in the image below. One can use the image `docker pull ghcr.io/knative/knative-docs:latest`
img

**Comment by csantanapr** [comment]
great !!! ❤️ 
/lgtm
/approve

**Pull request merged**